build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-rc1'
```

Comment:
It's not a good idea to use gradle in not final versions, 3.0.0-rc1 is a release candidate.